powerauth-push-server/src/main/java/io/getlime/push/service/fcm/model/MessageWithValidation.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.push.service.fcm.model;
+
+import com.google.firebase.messaging.Message;
+
+/**
+ * Model class for FCM requests with message and validation mode.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class MessageWithValidation {
+
+    private Message message;
+    private boolean validate_only;
```

Comment:
Let's use JSON annotation to assure the underscore-case, let's use cammel-case in Java.

Reply:
Ok, fixed.